src/stories/AmplienceImageStudio.stories.ts

```diff
@@ -0,0 +1,149 @@
+import { StoryObj, Meta } from '@storybook/html';
+import { expect } from '@storybook/test';
+import { FetchProvider, FileService, FileServiceResponse } from '@amplience/gql-ai-sdk';
+import {
+  AmplienceImageStudio,
+  AmplienceImageStudioOptions,
+  LaunchImageStudioOptions,
+} from '../AmplienceImageStudio';
+import { ImageStudioReason } from '../types';
+
+interface AmplienceImageStudioProps {
+  imageUrl: string;
+  provider: {
+    token: string;
+    serviceUrl: string;
+  }
+  options: AmplienceImageStudioOptions;
+}
+
+const meta: Meta<AmplienceImageStudioProps> = {
+  title: 'Amplience Image Studio',
+  tags: ['autodocs'],
+  argTypes: {},
+  render: () => {
+    return document.createElement('div');
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+export const TryMe: Story = {
+  args: {
+    provider: {
+      token: 'Bearer-Token-Here',
+      serviceUrl: 'https://graphql-gateway.amplience-qa.net/graphql',
+    },
+    imageUrl:
+      'https://thumbs.amplience-qa.net/r/a124da68-5b5d-46a7-94dc-13a4c45976f8',
+    imageName: 'TestImageName',
+    options: {
+      baseUrl: 'https://qa-mandelbrot-image-studio.amplience-qa.net/image-studio/',
```

Comment:
Should this be this url? if no, its used in this file a few times

Reply:
qa-release was pointing to the older topic/shoptalk branch, and therefore was out of sync with the messaging contract that this SDK uses. This has been updated now.